diff --git a/node_modules/react-native-svg/android/build.gradle b/node_modules/react-native-svg/android/build.gradle
index 7551306..7b807a9 100644
--- a/node_modules/react-native-svg/android/build.gradle
+++ b/node_modules/react-native-svg/android/build.gradle
@@ -21,6 +21,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
+    namespace 'com.horcrux.svg'
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     defaultConfig {
@@ -28,6 +29,10 @@ android {
         //noinspection OldTargetApi
         targetSdkVersion safeExtGet('targetSdkVersion', 28)
     }
+    buildFeatures {
+        buildConfig true
+    }
+
     lintOptions {
         abortOnError false
     }
